src/components/navigation/right-nav.sass

```diff
@@ -110,7 +164,7 @@ $list-item-scale: 0.11
           .info
             margin-top: $half-margin
             text-align: center
-            font-size: 0.6em
+            font-size: 10px
```

Comment:
I notice that the project uses both em and px to specify fonts.  Michael's specs tend to use specify a straight pixel size so that is what I favored.